pkg/lib/cockpit-components-file-autocomplete.jsx

```diff
@@ -184,8 +193,10 @@ FileAutoComplete.propTypes = {
     id: PropTypes.string,
     placeholder: PropTypes.string,
     superuser: PropTypes.string,
+    fileExists: PropTypes.bool,
     onChange: PropTypes.func,
 };
 FileAutoComplete.defaultProps = {
+    fileExists: true,
```

Comment:
I find this property name a bit odd, but sure. ("mustExist" or invert it and "allowNotExist" ?)

Reply:
True, the property existed in the other component therefore I left it, but I will also would rather change it. I will repost.